lib/api/index.js

```diff
@@ -359,12 +360,27 @@ app.post('/:installationId/migrate/:undo?', [
 
   try {
     if (req.params.undo) {
-      await jiraClient.devinfo.migration.undo();
+      /**
+       * Only call github/migrationUndo in prod. Undo will only be called in Jira if
+       * GITHUB_CONNECT_APP_IDENTIFIER is equal to com.github.integration.production
+       */
+
+      if (isProd()) {
+        await jiraClient.devinfo.migration.undo();
+      }
```

Comment:
Sorry, I might not have been clear.  I meant doing a file called `jiraClient` or something along those lines and you could add the logic of 'isProd' in there so that out here, it just looks like you're doing `jiraClient.undo()` and it handles the logic by itself

Reply:
One way to do that is have our own file that wraps out the jira client so that you have a function of 'undo' which does the logic and then calls the real jiraClient, or you can override the actual `undo` function and add the logic in there and still calling the old function after the logic passes.